### DISCUSSION TITLE
Searching for lemmy groups no longer works

### DISCUSSION THREAD

**Discussion by joshleblanc** [mastodon/mastodon#25623] [Q&A] [answered]
I'm nearly positive I've just done something wrong, but I can't seem to find lemmy groups when searching anymore.

I'm hosting my own instance, and I spun it up quite awhile ago and only recently updated to the latest master. It could be I'm missing a new environment variable or something.

![image](https://github.com/mastodon/mastodon/assets/1729810/44336f05-c66b-4740-b931-5208250d8ad6)

I can't even search for groups I'm actively following:

![image](https://github.com/mastodon/mastodon/assets/1729810/b26d1900-9a9e-465e-b458-d785d4adf592)

![image](https://github.com/mastodon/mastodon/assets/1729810/8eb35f5c-a8a5-4a44-8b08-29bd713d5fb2)

**Comment by wakest** [comment]
@joshleblanc it seems to be an issue with lemmy on the newest 0.18 version

**Comment by Cysioland** [comment]
I have the same issue. Can search for users but not groups.

**Comment by wakest** [comment, marked as answer]
this has been fixed along time ago. the issue was on the Lemmy side.
